### ISSUE TITLE
[automatically] deploy to rubygems

### ISSUE THREAD

**Issue by igalic** [voxpupuli/voxpupuli-release#4]
we should setup a pipeline to automagickally deploy to rubygems
related: https://github.com/voxpupuli/plumbing/issues/20

**Comment by danzilio** [comment]
should we even bother deploying to rubygems? we can probably just get away with pulling it from github

**Comment by igalic** [comment]
yeah, i think deploying to rubygems might be a [good idea](https://blag.esotericsystems.at/articles/automation/). As someone extraordinarily smart writes:

> * Put packages into repositories
> 
> This makes installation actually easy, and dependency management possible within the above mentioned atomic transaction.
> 
> Consider providing your repositories over IPv6…
>
> Oh, your package repository should not be (only) github.

**Comment by rnelson0** [comment]
I want to use it in my Gemfile and think others might as well.

We'll need an account first, then I think we can grant other people with existing accounts with the ability to create new versions.

**Comment by danzilio** [comment]
You can use it in a Gemfile from a git source. I would normally agree about shipping it to RubyGems, but I don't see much added benefit since this is really just an internal tool. But I'll defer to the majority.

**Comment by rnelson0** [comment]
Oh, but I want to use it externally! It's a pretty cool tool and could help other module authors, too.

**Comment by rnelson0** [comment]
When we have an account and so can get an API token, https://docs.travis-ci.com/user/deployment/rubygems/
